packages/toolpad-app/src/toolpadDataSources/server.ts

```diff
@@ -4,19 +4,21 @@ import postgres from './postgres/server';
 import rest from './rest/server';
 import googleSheets from './googleSheets/server';
 import movies from './movies/server';
-import config from '../config';
+
+import config from '../server/config';
 
 type ServerDataSources = { [key: string]: ServerDataSource<any, any, any> | undefined };
 
 const serverDataSources: ServerDataSources = config.isDemo
```

Comment:
Would it make sense to unify this behavior, and the `isDemoFeature` flag into 1 constant under `src/constants.tsx`?
```ts
export const DEMO_DATASOURCES = new Set(['rest', 'function', 'movies'])
export const PRODUCTION_DATASOURCES = new Set(['rest', 'function', 'googleSheets', 'postgres'])
```

Reply:
Sounds good, it's a bit redundant right now.